src/metabase/analyze/classifiers/name.clj

```diff
@@ -184,7 +184,7 @@
    [(prefix-or-postfix "companies")    :entity/CompanyTable]
    [(prefix-or-postfix "vendor")       :entity/CompanyTable]])
 
-(mu/defn infer-entity-type :- analyze.schema/Table
+(mu/defn infer-entity-type-by-name :- analyze.schema/Table
```

Comment:
I renamed this because `classifiers.name/infer-entity-type` clearly implied this was based on column name, however `analyze/infer-entity-type` lost that clarity